src/services/laodeai/handlers/zeroclick.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Process Github Gist from Cheerio readout
+ * @param {import('cheerio').CheerioAPI} $
+ * @returns {{ type: 'text' | 'error', content: String}}
+ */
+
+export function zeroclick($) {
+  const zeroclick = $('.zci__result').html();
+  return zeroclick
+    ? {
+        type: 'text',
+        content: zeroclick.trim().replace(/^\s+/gm, ''),
```

Comment:
add another replace for \r\n to \n will you? thanks.

Reply:
tadi udah replace gitu tapi masih fail testnya, hmm, coba lagi deh